check/nbformat

```diff
@@ -30,21 +30,29 @@ cd "$(git rev-parse --show-toplevel)"
 
 # Install a pinned version of tf-docs
 TF_DOCS_COMMIT=b3dc8a922d8bdc6e998a8ad4f4953359dc6e576a
-TF_DOCS_INSTALLED=$(pip list | grep  -c $TF_DOCS_COMMIT)
+TF_DOCS_INSTALLED="$(pip list | grep -c "$TF_DOCS_COMMIT")"
+BLACK_INSTALLED="$(pip list | grep -c 'black-nb')"
```

Comment:
I'd rather have this as a dependency added to dev_tools/conf/pip-list-dependencies.txt